### PR TITLE
[css-fonts] Add initial rendering tests for font-palette

### PR DESCRIPTION
**Pull request by litherum**
This test adds a test font which has one COLR glyphs with 2 colors, and a CPAL table with 2 palettes.

### CODE REVIEW

**Review by drott — COMMENTED** [review]
Thanks for your efforts covering font-palette with tests. A really important addition to color font usage, which I also aim to implement later this year. 

Comments below.

File: css/css-fonts/font-palette-ref.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+<title>CSS Test: font-palette drawing</title>
+<link rel="author" title="Myles C. Maxfield" href="mmaxfield@apple.com">
+<link rel="match" href="font-palette-ref.html">
+<link rel="help" href="https://drafts.csswg.org/css-fonts-4/#font-palette-prop">
+<meta name="assert" content="Font palettes are applied as necessary">
+</head>
+<body>
+<p>Test passes if you see particular rectangles of particular colors.</p>
+<div style="position: relative;">
+<div style="position: absolute; left: 0px; top: 0px; width: 100px; height: 100px; background: #FF00FF;">
```
[the remark this answers is not in the document]
Comment (litherum) [reply css/css-fonts/font-palette-ref.html:14]: Yeah, you're right. I'll update the font. I'll also add `light` and `dark` palettes too, and named palettes/colors, so those can be tested.

[the remark this answers is not in the document]
Comment (drott) [reply css/css-fonts/font-palette-ref.html:14]: Sounds good, thanks.

**Comment by litherum** [comment]
Closed in favor of https://github.com/web-platform-tests/wpt/pull/31044

**Pull request closed without merging**